### PR TITLE
Adding documentation for rerouting timeouts

### PR DESCRIPTION
**Pull request by WojcikMike**
@Particular/docs-maintainers please have a look as I am not entirely sure if I used partials correcly and if it is the place that this content should reside in.

### CODE REVIEW

**Comment by WojcikMike** [comment]
This doco was inspired by @andreasohlund pushing this: https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/particularsoftware/L3VXLbSzO28/aMR163a3CAAJ
On my plate.

File: nservicebus/ravendb/reroute-existing-timeouts.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Rerouting Existing Timeouts
```
Comment (SimonCropp) [inline nservicebus/ravendb/reroute-existing-timeouts.md]: u will need a component here

Comment (SimonCropp) [inline nservicebus/ravendb/reroute-existing-timeouts.md]: can u add a reviewed header

File: nservicebus/ravendb/reroute-existing-timeouts.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+ - RavenDB
+ - Persistence
+ - Timeouts
+---
+
+## Rerouting existing timeouts 
+
+After moving endpoint from one machine to another or changing endpoint name existing timeouts will need to be manually modified so that they could be delivered to new endpoint. In order to do that the following steps should be followed:
```
Comment (SimonCropp) [inline nservicebus/ravendb/reroute-existing-timeouts.md]: "After moving an endpoint"

**Comment by andreasohlund** [comment]
>This doco was inspired by @andreasohlund

Muuuhaahaahaa

**Comment by WojcikMike** [comment]
@SimonCropp I saw your comments, yet I have a higher level question should this exist as a stand alone document or be a part of 1 of the existing pages?

**Comment by SimonCropp** [comment]
@WojcikMike TBH the raven docs need a re-org. leave the PR as is and i will try to get the re-org done tomorrow

**Comment by WojcikMike** [comment]
@SimonCropp I addressed your initial comments. The big question stand. As we might move it even away from Raven and keep it as a general note. Or copy-paste it into other persistence as well.

**Comment by SimonCropp** [comment]
> The big question stand. As we might move it even away from Raven and keep it as a general note. 

in that case perhaps make it a partial?

**Comment by WojcikMike** [comment]
@SimonCropp 
> in that case perhaps make it a partial?

Can I have a partial.partial? As my screen use partials to capture differences between v5 and v6 of core

**Comment by SimonCropp** [comment]
@WojcikMike sorry i meant an include https://github.com/Particular/docs.particular.net#markdown-includes

**Comment by WojcikMike** [comment]
@Particular/ravendb-persistence-maintainers @timbussmann any advice on when this piece of information should live? For now I added it as a separate site on Raven.

@SimonCropp I am not going with include for now. As I struggle to find place for this info, and I would like to get some support in placing this thing.

**Comment by weralabaj** [comment]
@WojcikMike I assume the problem is not raven-specific, or is it?

Based on the above assumption IMO you could do it the following way:
- Add a high-level article on decomissioning an endpoint, to provide some background for the situation. For now it can be very short and only describe the scenario. That article should be included https://docs.particular.net/nservicebus/endpoints/ and https://docs.particular.net/nservicebus/operations/ and refer to the "technology-specific" articles for more details. i.e. in this case only your new article for now.
- Leave the article you created where it is now, but refer back to the decomissioning article to provide "bigger picture".

I suggest doing that because I wasn't sure what's the context for this before I read the GG thread, so I feel we need a bit more "description". If you need help and like the proposition ping me.

**Comment by WojcikMike** [comment]
@weralabaj please have a look if this is something that you had in mind

**Comment by weralabaj** [comment]
I like it! Ready for review? If not ping me when I should have a look.

**Comment by WojcikMike** [comment]
@weralabaj go for it, in the meantime let me look at manu file as I messed up something

**Pull request merged**